extensions/mongoProcessor/MongoQueueProcessor.js

```diff
@@ -90,12 +137,26 @@ class MongoQueueProcessor {
             });
             this._consumer.on('ready', () => {
                 consumerReady = true;
-                this._consumer.subscribe();
+                const paused = options && options.paused;
+                this._consumer.subscribe(paused);
                 this.logger.info('mongo queue processor is ready');
             });
         });
     }
 
+    getServiceVars() {
+        return {
+            logger: this.logger,
+            site: this.site,
+            zkClient: this.zkClient,
+            consumer: this._consumer,
```

Comment:
Thinking out loud here: since we are exporting this, maybe we could remove the underscore. I don't think we've done that consistently, but that seems to make sense to me. 😸